src/nvim/api/autocmd.c

```diff
@@ -301,40 +327,68 @@ Array nvim_get_autocmds(Dict(get_autocmds) *opts, Error *err)
+///     })
+/// </pre>
 ///
-/// @param event The event or events to register this autocmd
-///          Required keys:
-///              event: string | ArrayOf(string)
+/// Example using command:
+/// <pre>
+///     vim.api.nvim_create_autocmd({
+///       event = {"BufEnter", "BufWinEnter"},
+///       pattern = {"*.c", "*.h"},
+///       command = "echo 'Entering a C or C++ file'",
+///     })
+/// </pre>
 ///
-///              Examples:
-///                 - event: "pat1,pat2,pat3",
-///                 - event: "pat1"
-///                 - event: { "pat1" }
-///                 - event: { "pat1", "pat2", "pat3" }
+/// Example values for pattern:
+/// <pre>
+///   pattern = "*.py"
+///   pattern = { "*.py", "*.pyi" }
+/// </pre>
 ///
-/// @param opts Optional Parameters:
-///         - callback: (string|function)
-///             - (string): The name of the viml function to execute when triggering this autocmd
-///             - (function): The lua function to execute when triggering this autocmd
-///             - NOTE: Cannot be used with {command}
-///         - command: (string) command
-///             - vimscript command
-///             - NOTE: Cannot be used with {callback}
-///                  Eg. command = "let g:value_set = v:true"
-///         - pattern: (string|table)
-///             - pattern or patterns to match against
-///             - defaults to "*".
-///             - NOTE: Cannot be used with {buffer}
-///         - buffer: (bufnr)
-///             - create a |autocmd-buflocal| autocmd.
-///             - NOTE: Cannot be used with {pattern}
-///         - group: (string|int) The augroup name or id
-///         - once: (boolean) - See |autocmd-once|
-///         - nested: (boolean) - See |autocmd-nested|
-///         - desc: (string) - Description of the autocmd
+/// Examples values for event:
+/// <pre>
+///   event = "BufPreWrite"
+///   event = {"CursorHold", "BufPreWrite", "BufPostWrite"}
+/// </pre>
+///
+/// @param event (String|Array) The event or events to register this autocommand
+/// @param opts Dictionary of autocommand options:
+///             - group (string|integer) optional: the autocommand group name or
+///             id to match against.
+///             - pattern (string|array) optional: pattern or patterns to match
+///             against |autocmd-pattern|.
+///             - buffer (integer) optional: buffer number for buffer local autocommands
+///             |autocmd-buflocal|. Cannot be used with {pattern}.
+///             - desc (string) optional: description of the autocommand.
+///             - callback (function|string) optional: Lua function or Vim function (as string) to
+///             execute on event. Cannot be used with {command}
+///             - command (string) optional: Vim command to execute on event. Cannot be used with
+///             {callback}
+///             - once (boolean) optional: defaults to false. Run the autocmd
```

Comment:
```suggestion
///             - once (boolean) optional: defaults to false. Run the autocommand
```

Reply:
My regex skills in shambles, missed one 🥲😁

Reply:
Pesky end-of-line, always messing things up ;)